File: javascript/1115.js
var input = require('fs').readFileSync('/dev/stdin', 'utf8');
var lines = input.split('\n');

while(lines.length){
    [x,y]=lines.shift().trim().split(' ').map((el) => parseInt(el))
    
    if(x==0 || y==0)break
    
    if(x>0){
        if(y>0)console.log('primeiro')
        else console.log('quarto')
    }else if(y>0){
        console.log('segundo')
    }else{
        console.log('terceiro')
    }
}

File: python/1115.py
while True:
    x,y=[int(el) for el in input().strip().split(' ')]
    if x==0 or y==0:
        break
    
    if x>0:
        if y>0:
            print("primeiro")
        else:
            print("quarto")
    elif y>0:
        print("segundo")
    else:
        print("terceiro")